cfme/tests/physical_infrastructure/api/test_physical_switch_inventory_api.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.rest import assert_response
+
+pytestmark = [
+    pytest.mark.tier(3),
+    pytest.mark.provider([LenovoProvider])
+]
+
+
+@pytest.fixture(scope="module")
+def physical_switch(setup_provider_modscope, appliance):
```

Comment:
Since this fixture is duplicated in both test modules, I think you should add a new `cfme/fixtures/physical_switch.py` module, with this fixture definition in it.  Add this module name to 'pytest_plugins' list in `cfme/test_framework/pytest_plugin.py`, and you'll be able to use the fixture in any test without having to import or define it locally - just add it to the method spec the same as it is now with the fixture defined locally.